content/docs/command-reference/exp/init.md

```diff
@@ -11,7 +11,7 @@ Quickly setup any project to use [DVC Experiments].
 usage: dvc exp init [-h] [-q | -v] [--run] [--interactive] [-f]
                     [--explicit] [--name NAME] [--code CODE]
                     [--data DATA] [--models MODELS] [--params PARAMS]
-                    [--metrics METRICS] [--plots PLOTS] [--live LIVE]
```

Comment:
This seems to depend on https://github.com/iterative/dvc/issues/7534, or else there is no way to configure the paths when using `--type=dl`.

Reply:
Opened https://github.com/iterative/dvc/pull/7603 and updated the description here. Not 100% sure if the description is understable